mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS wv net/minecraft/resource/ObjectPassingResourceReloadListener
+	METHOD a apply (Ljava/lang/Object;Lwt;Lafm;)V
```

Comment:
Not sure about `apply` here, imo `accept` works better. Java stdlib has `Consumer#accept` and `Function#apply`, of which `accept` matches better.

Reply:
I'm referring to the "load" and "apply" stages of resource reloading. The canon names are "prepare" and "apply", FWIW.

Reply:
But we can discuss "load" -> "prepare" later.